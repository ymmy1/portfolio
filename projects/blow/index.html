<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<link rel="stylesheet" href="main.css">
		<link rel="stylesheet" href="responsive.css">
		<title>BLOW</title>
	</head>
	<body>
		<header>
			<div class="top">
				<a href="index.html" class="logo">blow</a>
				<nav>
					<ul>
						<li><a href="#">home</a></li>
						<li><a href="#">about</a></li>
						<li><a href="#">work</a></li>
						<li><a href="#">contact</a></li>
					</ul>
				</nav>
			</div>
			<div class="side">
				<div class="side-text">
					<p class="first-part">get in touch</p>
					<p class="second-part">have project in mind</p>
				</div>
				<a href="#"><img src="img/icon1.png" alt=""></a>
			</div>
			<div class="btn">
				<button class="main">explore work</button>
			</div>
			
		</header>
	</body>
</html>