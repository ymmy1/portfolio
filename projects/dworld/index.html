<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title>Different World</title>
		<link rel="shortcut icon" type="image/x-icon" href="img/title-icon.png">
		<link rel="stylesheet" href="styles/main.css">
	</head>
	<body>
		<header>
			<div class="container">
				<div class="menu"><span><a href="index.html" class="logo">Different World</a></span>
				<button onclick="window.location.href='https://www.tesla.com/'" class="style-scope">Skip the content and buy electric car</button>
				<button class="style-scope">Call to our 24/7 support and let us do all the work</button>
				<nav>
					<ul>
						<li><a href="menu.html">Menu</a></li>
						<li><a href="#">Contacts</a></li>
						<li><a href="#">About Us</a></li>
					</ul>
				</nav>
			</div>
			<section class="main"><h1>Swap your car to electric one</h1>
				<p>Clean the air you breath.</p>
				<div class="buttons">
					<a href="https://www.tesla.com/" class="find button">Find Electric Car</a>
					<a href="https://www.cashautosalvage.com/" class="junk button"> Junk your old one</a>
				</div>
				<p class="disc">Have a 10% discount for bringing a friend</p>
			</section>
		</div>
	</header>
	<footer>
		<h2>Sponsors</h2>
		<div class="sponsors">
			<img src="img/sponsors.png" alt="Sponsor number 1. Temproary the Earth's logo">
			<img src="img/sponsors.png" alt="Sponsor number 2. Temproary the Earth's logo">
			<img src="img/sponsors.png" alt="Sponsor number 3. Temproary the Earth's logo">
		</div>
	</footer>
</body>
</html>