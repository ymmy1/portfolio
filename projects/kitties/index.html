<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>KITTIES TO GO</title>
	<link rel="stylesheet" href="main.css">
</head>
<body>
	<header>
		<div class="box">
			<section class="title"><h1>Kitties to go!</h1>
		<p class="header">Seattle’s premiere drive-thru
			cat rentals</p>
		</section>
		</div>
	</header>
	<nav>
	<div class="box">
		<ul>
			<li><a href="#">About</a></li>
			<li><a href="#">Rates</a></li>
			<li><a href="#">Liability Release</a></li>
			<li><a href="#">Kitty Profiles</a></li>
			<li><a class="nav" href="#">Book a Cat</a></li>
		</ul>
	</div>
	</nav>
	<div class="box">
	<section class="main">
	<article>
		<h3>Cat ipsum dolor sit amet, find something else more interesting, so catty ipsum mesmerizing birds sleep on my human’s head.</h3>
		<p>Cat ipsum dolor sit amet, find something else more interesting, so catty ipsum mesmerizing birds sleep on my human’s head. <span>Hide head under blanket</span> so no one can see stares at human while pushing stuff off a table. Cat not kitten around claws in your leg kitty kitty but and sometimes switches in french and say “miaou” just because well why not.</p>
		<p class="jeremy">Please stop looking asat your phone an pet me. Lounge in the doorway eat and then sleep on your face meowwww meowwww – Jeremy</p>
		<h2>The Kitties to go story</h2>
		<p>Fall asleep on the washing machine snuggles up to shoulders or knees and purrs you to sleep. Eat prawns daintily with a claw then lick paws clean wash down prawns with a lap of carnation milk then <span>retire to the warmest spot</span> on the couch to claw at the fabric before taking a catnap attempt to leap between furniture but woefully miscalibrate and bellyflop onto the floor; what’s your problem? i meant to do that now i shall wash myself intently cough and swat at dog.</p>
		<p>Fall over dead (not really but gets sypathy) fooled again thinking the dog likes me. Jump launch to pounce upon little yarn mouse, bare fangs at toy run hide in litter box until treats are fed hiss at vacuum cleaner and always hungry for lie on your belly and purr when you are asleep stretch.</p>
	</article>
	<aside>
		<h2>Cats in news</h2>
		<p class="article">Chew iPad power cord why must they do that, grab pompom in mouth and put in water dish.</p>
		<a class="article-link" href="#">Find out more >></a>
		<p class="article2">Chew iPad power cord why must they do that, grab pompom in mouth and put in water dish.</p>
		<a class="article-link2" href="#">Find out more >></a>
		<h2 class="cat-week">Cat of the week</h2>
		<figure>
			<img src="img/sidebar-cat.jpg" alt="">
			<figcaption>Stare at imaginary bug love blinks and purr purr purr purr</figcaption>
		</figure>	
			<a class="article-link" href="#">Book now >></a>
	</aside>
	</section>
	</div>
	<footer>
		<div class="box">
		<img src="img/cat-logo.png" alt="">
		<p>Aborpos nisto inci tet aut que volupti beruptiberro to ex<br>
		esseque invenienihil imaximu scideru ptistiusam consequunt</p>
		<ul class="footer">
			<li><a class="f-link" href="#">Footer link 1</a></li>
			<li><a class="f-link" href="#">Footer link 2</a></li>
			<li><a class="f-link" href="#">Footer link 3</a></li>
			<li><a class="f-link" href="#">Footer link 4</a></li>
		</ul>
		<p>© 2018 Kitties To Go!, LLC | Seattle, WA</p>
	</div>
	</footer>
	
</body>
</html>