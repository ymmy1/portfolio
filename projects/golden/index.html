<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<title>Golden</title>
		<link rel="stylesheet" href="css/main.css">
	</head>
	<body>
		<header>
			<div class="container">
				<div class="heading clearfix">
					<img src="img/logo.png" alt="Golden" class="logo">
					<nav>
						<ul class="menu">
							<li> <a href="#">Home</a></li>
							<li> <a href="#">Services</a></li>
							<li> <a href="#">Portfolio</a></li>
							<li> <a href="#">About</a></li>
							<li> <a href="#">Contacts</a></li>
						</ul>
					</nav>
				</div>
				
				<div class="titles">
					<div class="titles_first">Welcome To Our Studio!</div>
					<h1>It’s nice to Meet you</h1>
				</div>
				<a href="#" class="button">Tell me more</a>
			</div>
		</header>
		<section id="services">
			<div class="container">
				<div class="title">
					<h2>Services</h2>
					<p>Proin iaculis purus consequat sem cure.</p>
				</div>
				<div class="services clearfix">
					<div class="services_item">
						<img src="img/icon1.png" alt="">
						<h3>E-Commerce</h3>
						<p>
							Proin iaculis purus consequat sem cure digni 	ssim. Donec porttitora entum suscipit
							aenean rhoncus posuere odio in tincidunt.
						</p>
					</div>
					<div class="services_item">
						<img src="img/icon2.png" alt="">
						<div class="text_lft">
							<h3>Responsive Web</h3>
							<p>
								Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
								aenean rhoncus posuere odio in tincidunt.
							</p>
						</div>
					</div>
					<div class="services_item">
						<div class="text_lft">
							<img src="img/icon3.png" alt="">
							<h3>Web Security</h3>
							<p>
								Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit
								aenean rhoncus posuere odio in tincidunt.
							</p>
						</div>
					</div>
				</div>
				
			</div>
		</section>
		<section id="portfolio">
			<div class="container">
				<div class="title">
					<h2>Our portfolio</h2>
					<p>Proin iaculis purus consequat sem cure.</p>
				</div>
				<div class="works clearfix">
					<img src="img/img1.jpg" alt="">
					<img src="img/img2.jpg" alt="">
					<img src="img/img3.jpg" alt="">
				</div>
			</div>
		</section>
		<section id="about">
			<div class="container">
				<div class="title">
					<h2>About us</h2>
					<p>Proin iaculis purus consequat sem cure.</p>
				</div>
				<div class="about clearfix">
					<div class="about_item_lft">
						<img src="img/about1.jpg" alt="">
						<h4>July 2010</h4>
						<h3>Our Humble Beginnings</h3>
						<p>Proin iaculis purus consequat sem cure<br>
							digni ssim. Donec porttitora entum suscipit<br>
							aenean rhoncus posuere odio in tincidunt. Proin<br>
							iaculis purus consequat sem cure digni<br>
						ssim. Donec porttitora entum suscipit.</p>
					</div>
					<div class="about_item_rgt">
						<img src="img/about2.jpg" alt="">
						<h4>January 2011</h4>
						<h3>Facing Startup Battles</h3>
						<p>Proin iaculis purus consequat sem cure<br>
							digni ssim. Donec porttitora entum suscipit<br>
							aenean rhoncus posuere odio in tincidunt. Proin<br>
							iaculis purus consequat sem cure digni<br>
						ssim. Donec porttitora entum suscipit aenean<br>rhoncus posuere odio in tincidunt.</p>
					</div>
					<div class="about_item_lft">
						<img src="img/about3.jpg" alt="">
						<h4>December 2012</h4>
						<h3>Enter The Dark Days</h3>
						<p>Proin iaculis purus consequat sem cure<br>
							digni ssim. Donec porttitora entum suscipit<br>
							aenean rhoncus posuere odio in tincidunt. Proin<br>
						iaculis purus consequat sem cure digni.</p>
					</div>
					<div class="about_item_rgt">
						<img src="img/about4.jpg" alt="">
						<h4>February 2014</h4>
						<h3>Our Triumph</h3>
						<p>Proin iaculis purus consequat sem cure<br>
							digni ssim. Donec porttitora entum suscipit<br>
							aenean rhoncus posuere odio in tincidunt. Proin<br>
							iaculis purus consequat sem cure digni<br>
							ssim. Donec porttitora entum suscipit aenean.
						</div>
					</div>
				</div>
			</section >
			<footer>
				<div class="container">
					<p>© Copyright 2014 FreebiesXpress.com</p>
					
				</div>
			</footer>
			
		</body>
	</html>